src/utils/triton.h

```diff
@@ -212,6 +254,77 @@ class TritonRequestView : public TritonRequestWrapper<TritonRequestView> {
   TRITONBACKEND_Request *handle_ = nullptr;
 };
 
+
+template<class Actual>
+class TritonResponseWrapper {
+ public:
+  OBufferDescr AllocateOutputBuffer(const IOMeta &out_info, device_type_t preferred_device) {
```

Comment:
There's a notable lack of symmetry in this API:
The request returns TritonInput object and the buffer allocation happens in TritonInput::GetBuffer. Here these steps are fused - Perhaps there should be a TritonOutput class?

Reply:
Created TritonOutput